pep-0394.txt

```diff
@@ -175,14 +262,17 @@ making such a change.
   versa. That way, if a sysadmin does decide to replace the installed
   ``python`` file, they can do so without inadvertently deleting the
   previously installed binary.
-* If the Python 2 interpreter becomes uncommon, scripts should nevertheless
-  continue to use the ``python3`` convention rather that just ``python``. This
-  will ease transition in the event that yet another major version of Python
-  is released.
+* Even as the Python 2 interpreter becomes less common, scripts should
```

Comment:
The "should" wording here doesn't match the revised weaker wording above. I'd suggest:

> Even as the Python 2 interpreter becomes less common, it remains reasonable for scripts to continue to use the `python` convention, rather than just `python`. In the event ...

Reply:
use the python convention, rather than just python? I don't quite understand that bit.

Reply:
Sorry, typo - I left out a critical "3" in the original version of the comment :)